src/test/java/org/broadinstitute/hellbender/utils/read/ArtificialReadUtilsUnitTest.java

```diff
@@ -2,8 +2,11 @@
 
```

Comment:
This file might have conflicts after https://github.com/broadinstitute/gatk/pull/3475

Reply:
Already addressed.  Done